main.tf

```
@@ -212,6 +212,13 @@ resource "google_pubsub_subscription" "push_subscriptions" {
       x-goog-version = each.value.minimum_backoff != null ? "x-goog-version" : "v1"
     }
 
+    dynamic "no_wrapper" {
+      for_each = (lookup(each.value, "no_wrapper", true) ? [true] : [])
+      content {
+        write_metadata = lookup(each.value, "write_metadata", true)
```

Should never fall back to default, but still might want to set to `null`?

```suggestion
        write_metadata = lookup(each.value, "write_metadata", null)
```